backend/controllers/DashboardController.php

```diff
@@ -31,8 +31,16 @@ public static function updateItem($id){
          ShopModel::updateItem($id);
          self::showIndexPage("Artikel številka $id je bil posodobljen!");
     }
-     public static function deleteItem($id){
-        ShopModel::deleteItem($id);
-        self::showIndexPage("Artikel je bil izbrisan!");
-     }
+    public static function deleteItem($id){
+       ShopModel::deleteItem($id);
+       self::showIndexPage("Artikel je bil izbrisan!");
+    }
+
+    public static function verifyUser() {
+       $certifikati = ProfileModel::getCertifikati();
+       $client_cert = filter_input(INPUT_SERVER, "SSL_CLIENT_CERT");
+       $cert_data = openssl_x509_parse($client_cert);
+       $commonname = $cert_data['subject']['CN'];
```

Comment:
```suggestion
       $commonname = $cert_data['subject']['CN'];
       $hashCommonName = .... // hash od $cert_data['subject']['CN'];
```
[https://www.php.net/manual/en/function.hash.php](url)